### ISSUE TITLE
atropos version 1.1.2 on conda hangs while pip version works

### ISSUE THREAD

**Issue by cokelaer** [jdidion/atropos#12]
For your information:

```
conda install atropos
```
installs atropos and tqdm packages as expected but, a call to atropos hangs forever on a small data sets. A control+C interruption gives this error:

```
2017-05-01 18:29:26,569 INFO: Starting 2 worker processes
2017-05-01 18:29:26,572 ERROR: Unknown error
Traceback (most recent call last):
  File "/home/cokelaer/miniconda3/envs/py3/lib/python3.5/site-packages/atropos/util/__init__.py", line 673, in run_interruptible
    func(*args, **kwargs)
  File "/home/cokelaer/miniconda3/envs/py3/lib/python3.5/site-packages/atropos/commands/multicore.py", line 291, in __call__
    self.ensure_alive)
  File "/home/cokelaer/miniconda3/envs/py3/lib/python3.5/site-packages/atropos/commands/multicore.py", line 507, in enqueue_all
    for item in iterable:
TypeError: iter() returned non-iterator of type 'tqdm'
```

Then, I tried the pip version:
```
conda remove atropos 
conda remove tqdm
pip install atropos
```

This does not seem to install tqdm and then running the previous atropos command gives the expected output files.

This was on a python 3.5 conda environment.

**Comment by jdidion** [comment]
Thanks Thomas. As far as the pip issue, tqdm is an “extra” meaning you have to run “pip install atropos[tqdm]” for that dependency to install.

I am looking into the first issue.

**Comment by jdidion** [comment]
Fixed in fcb594b. I will make a new release for this soon.
